cmd/controller/app/options/options.go

```diff
@@ -183,6 +191,8 @@ func (s *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
 		"will be attempted.")
 	fs.StringVar(&s.Kubeconfig, "kubeconfig", defaultKubeconfig, ""+
 		"Paths to a kubeconfig. Only required if out-of-cluster.")
+	fs.Float32Var(&s.KubernetesAPIQPS, "kubernetes-api-qps", defaultKubernetesAPIQPS, "indicates the maximum queries-per-second requests to the Kubernetes apiserver")
+	fs.IntVar(&s.KubernetesAPIBurst, "kubernetes-api-burst", defaultKubernetesAPIBurst, "the maximum burst queries-per-second of requests sent to the Kubernetes apiserver")
```

Comment:
Looking at the kube-controller-manager flags here: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/

1) `--kube-api-qps` and `--kube-api-burst`
2) It seems like the controller manager actually already overrides these values to be higher.. perhaps it _is_ worth changing the defaults too 🤔

Reply:
will change the flags
and yeah... i found 5 quite low...